packages/lodestar/src/opPool/opPool.ts

```diff
@@ -50,7 +50,7 @@ export class OpPool extends EventEmitter {
     this.db = db;
     this.proposers = new Map();
     this.attestations = new AttestationOperations(this.db.attestation, {config});
-    this.aggregateAndProofs = new AggregateAndProofOperations(this.db.aggregateAndProof);
+    this.aggregateAndProofs = new AggregateAndProofOperations(this.db.aggregateAndProof, {config});
```

Comment:
Should we remove old AggregateAndProofs in `processBlockOperations`?

Reply:
oh yeah, forgot to change that, we should preserve attestations but remove included aggregates